cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -683,36 +670,6 @@ SwiftGenerator::validateSwiftModuleMappings(const UnitPtr& unit)
                << current->second << "'" << endl;
```

Comment:
I would keep the message on a single line, no \n.

Reply:
I didn't even notice this was split up. Fixed.